asapdiscovery-ml/asapdiscovery/ml/utils.py

```diff
@@ -250,6 +258,8 @@ def build_model(
             config = {}
     elif (type(config) is str) or isinstance(config, Path):
         config = parse_config(config)
+    elif type(config) is not dict:
```

Comment:
Long term may be an idea to move these configs to .... you guessed it, schema. But no need in this PR.

Reply:
yep that's a good idea. opened #450 to track this